scripts/apps/authoring/authoring/services/AuthoringWorkspaceService.js

```diff
@@ -44,6 +44,12 @@ export function AuthoringWorkspaceService($location, superdeskFlags, authoring,
             }
         }.bind(this);
 
+        // disable open for external ingest sources that are not editable(fetch not available)
+        if (item._type === 'externalsource' && config.features && 'editFeaturedImage' in config.features &&
```

Comment:
This should also work as:
```js
(item._type === 'externalsource' && !config.features.editFeaturedImage)
```
`config.features` is always there as far as I know, it defaults as object. See [here](https://github.com/superdesk/superdesk-client-core/blob/master/webpack.config.js#L178). Much shorter and clearer imo.